src/frequenz/client/assets/_client.py

```diff
@@ -138,3 +137,47 @@ async def list_microgrid_electrical_components(
         return [
             electrical_component_proto(component) for component in response.components
         ]
+
+    async def list_microgrid_electrical_component_connections(
+        self,
+        microgrid_id: int,
+        source_component_ids: list[int] | None = None,
+        destination_component_ids: list[int] | None = None,
```

Comment:
You could use this instead to make it more generic and even simplify the code:


```suggestion
        source_component_ids: Iterable[int] = (),
        destination_component_ids: Iterable[int] = (),
```